templates/add_data.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'layout.html' %}
+
+{% block body %}
+    <h1>Add data Modules</h1>
+    <small>Welcome {{session.name}}</small>
+    <div></div>
+    <a class="btn btn-success" href="/add_data">Add additional modules</a>
+    <hr>
+        <h2>Folder Number</h2>
+        <h2>{{patient.folder_number}}</h2>
+                <td><a href="add_biopsy/{{patient.folder_hash}}" class="btn btn-success">Add Biopsy</a></td>
```

Comment:
>add_biopsy [](start = 29, length = 10)

what if i already added it?

Reply:
then you would need to do edit_biopsy.. so maybe on the add_data page top will be add_x and bottom half is edit_x?